processors/p_process_lq_calibration_cut.jl

```diff
@@ -178,28 +178,64 @@ function p_process_lq_calibration_cut(processing_config::PropDict, l200::LegendD
                     throw(ErrorException("Error in energy correction and normalization: $e"))
                 end
 
-                @debug "Calibrate $lq_type"
+                @debug "Drift time correction of $lq_type"
                 drift_result, drift_report = nothing, nothing
                 try 
                     drift_result, drift_report = lq_ctc_correction(lq_e_corr, dt_eff, e_cal, dep_µ, dep_σ;
                     ctc_dep_edgesigma, ctc_lq_precut_relative_cut, lq_outlier_sigma, ctc_driftime_cutoff_method, dt_eff_outlier_sigma, lq_e_corr_expression, dt_eff_expression, ctc_dt_eff_low_quantile, ctc_dt_eff_high_quantile, pol_fit_order, uncertainty=ctc_uncertainty)
                 catch e
-                    @error "Error in drift time correction: $e"
-                    throw(ErrorException("Error in drift time correction: $e"))
+                    @error "Error in drift time correction: $(truncate_error(e))"
+                    throw(ErrorException("Error in drift time correction: $(truncate_error(e))"))
                 end
 
-                #create and save plots
+                # create and save plots
                 p = LegendMakie.lplot(drift_report, e_cal, dt_eff, lq_e_corr, :DEP, title = "Drift Time vs LQ in DEP for Detector: $det", figsize = (620,400))
                 savelfig(LegendMakie.lsavefig, p, l200, part, filekey_ch, det, Symbol("lq_ctc_DEP_$lq_type"))
 
                 p = LegendMakie.lplot(drift_report, e_cal, dt_eff, lq_e_corr, :whole, title = "Drift Time vs LQ for Detector: $det", figsize = (620,400))
                 savelfig(LegendMakie.lsavefig, p, l200, part, filekey_ch, det, Symbol("lq_ctc_$lq_type"))
 
-                #create log entry
+
+                # create lq normalization                
+                lq_ctc = nothing
+                lq_class_expression = nothing
+                result, report = nothing, nothing
+                lq_class = nothing
+                try
+                    lq_class_expression = drift_result.func
+                    lq_ctc = ljl_propfunc(lq_class_expression).(hit_cal)
+
+                    result, report = lq_norm(dep_µ, dep_σ, e_cal, lq_ctc; 
+                    dep_sideband_sigma, cut_truncation_sigma, uncertainty=cut_uncertainty, lq_class_expression)
+                    
+                    lq_class = ljl_propfunc(result.func).(hit_cal)
+                catch e
+                    @error "Error in LQ normalization calculation: $(truncate_error(e))"
+                    throw(ErrorException("Error in LQ normalization calculation: $(truncate_error(e))"))
+                end
```

Comment:
You overwrite `lq_class` in line 273 without ever using this before.
You might want to get rid of variables that are not used.
```suggestion
                # create lq normalization                
                lq_ctc = nothing
                lq_class_expression = nothing
                result, report = nothing, nothing
                try
                    lq_class_expression = drift_result.func
                    lq_ctc = ljl_propfunc(lq_class_expression).(hit_cal)

                    result, report = lq_norm(dep_µ, dep_σ, e_cal, lq_ctc; 
                    dep_sideband_sigma, cut_truncation_sigma, uncertainty=cut_uncertainty, lq_class_expression)
                catch e
                    @error "Error in LQ normalization calculation: $(truncate_error(e))"
                    throw(ErrorException("Error in LQ normalization calculation: $(truncate_error(e))"))
                end
```